### ISSUE TITLE
Move the database out of the app container

### ISSUE THREAD

**Issue by morrismukiri** [kahummer/AutoStandup#17]
You want your database out of the app container. 

You can do this by either attaching a volume to the app container that will hold your file-based database, or create a database container that persists the data.
I'd suggest using a separate database container all-together.

**Comment by ellykits** [comment]
Yes we will go with the latter but we can use the former for first deployment of MVP
